local_settings.ini

```diff
@@ -0,0 +1,11 @@
+# This file contains settings related to the local machine
+[config]
+output_dir=~/simout/quantum-in_situ_opt/skopt
+
+# default parameter file to be used. params-quant_self_opt.ini
+# for ArXiv paper settings use ncl-params-ArXiv1810.12178.ini
+param_file=params-quant_self_opt.ini
+
+verbosity=1
+use_tex_for_plots=True
```

Comment:
Not sure that this is implemented in this project. Is the property used anywhere?

Reply:
It isn't; whence, removed.